index.html

```diff
@@ -52,19 +52,19 @@
         </div>
     </div>
     <div id="videofull">
-        <video class="lazy" autoplay loop muted playsinline poster="/assets/NorthstarPromoPoster.jpg"><source src="/assets/NorthstarPromoVid.mp4"></video>
+        <video id="lazyvideo" class="lazy" autoplay loop muted playsinline poster="/assets/NorthstarPromoPoster.jpg"><source src=""></video>
         <div class="overlay"></div>
         <div id="mainbuttons">
             <div id="centerlogo">
-                <img class="logo" src="/assets/logo_1k.png" alt="Northstar Logo"/>
+                <img class="logo" src="/assets/logo_192.png" alt="Northstar Logo"/>
                 <span class="northstartext">NORTHSTAR</span>
             </div>
             <p class="northstardescription"><span class="tffont">titanfall<span class="tfdivider"></span> 2</span> modding and custom server framework</p>
             <div class="centerbuttons">
                 <a ondragstart="return false;" href="/discord" target="_blank" class="button big"><img src="/assets/icon_discord.svg" alt="Discord Server"/><span>Discord</span></a>
                 <a href="/wiki" class="button big"><img src="/assets/icon_wiki.svg" alt="Northstar Wiki"/>Wiki</a>
-                <a href="/github" class="button big"><img src="/assets/github.png" alt="Modding Documentation"/>GitHub</a>
-                <a href="/servers" class="button big"><img src="/assets/globe.png" alt="Modding Documentation"/>Server Browser</a>
+                <a href="/github" class="button big"><img src="" class="gh_img" alt="Github"/>GitHub</a>
+                <a href="/servers" class="button big"><img id="browser-globe" src="" alt="Server Browser"/>Server Browser</a>
```

Comment:
This goes for all, but class names should be expanded

Reply:
cringe